Source/Tests/TrackableEntities.Client.Tests/ChangeTrackingCollectionTests.cs

```diff
@@ -719,6 +719,28 @@ public void EntityChanged_Event_Should_Fire_When_Items_Added_Modified_Deleted()
             Assert.Equal(3, changesCount);
         }
 
+        [Fact]
+        public void EntityChanged_Event_Should_Fire_When_Graph_Items_Added_Modified_Deleted()
+        {
+          // Arrange
+          var changeCount = 0;
+
+          var database = new MockNorthwind();
+          var order = database.Orders[0];
+          var changeTracker = new ChangeTrackingCollection<Order> { order };
+          changeTracker.EntityChanged += (s, e) => changeCount++;
+          changeTracker.Tracking = true;
+
+          // Act
+          order.OrderDetails.Add(new OrderDetail());
+          order.Customer.CustomerName = "Tony sneed";
+          order.OrderDetails.RemoveAt(0);
+          order.OrderDetails[0].Product.Category.CategoryName = "Lorem ipsum";
```

Comment:
Just need to figure out why change to `Category` on line 738 does not fire the EntityChanged event.  See if we're not subscribing to it.